Task-56509: Configure the agenda Application as a main page application

Amine0008 opened the pull request:
This change will allow to capture the main application name. In fact, any application will be able to define the variable eXo.env.portal.mainApplicationName (with the application/module name) inside its required JS in order to add an additional data property inside collected operation pageFullUIDisplay

halaya committed:
TASK-35704: Create FB maintenance and update projects versions/dependencies

Amine0008 committed:
added the mainApplicationName property when creating the agenda application

boubaker commented:
Please update title & description as suggested in other PRs and add application name label inside Analytics_en.properties as suggested here https://github.com/Meeds-io/social/pull/1435#discussion_r858927118

IlhemEssaadi committed:
TASK-56372: ICS file in event invitation have not the right date (#427)

ISSUES : the startDate/endDate in ics file does not correspond with the dates of event.
FIX : the problem is that we generate the ics file without a time zone,this fix refactored the addIcsFile method to implement the ical4j library and use the user's time zone as a parameter (if the user is guest, we use the event's time zone instead of the user's time zone) to create the ics file with the correct dates.

azayati committed:
Task-55805: [Accessibility - Contrast - 3.2] Change the default primary color (#430)

Amine0008 committed:
changed the mainApplication name Property from Agenda to agenda so that the property can be translated
